pom.xml

```diff
@@ -1109,12 +1109,6 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.nifi</groupId>
-                <artifactId>nifi-kudu-nar</artifactId>
```

Comment:
Did you intend to remove nifi-kudu-nar?

Reply:
It was duplicated in the pom; Maven issued a warning to me about it.

Reply:
I understand.  I think that might make a great change in a different PR for a different ticket.  But again, I would prefer not to evaluate it as part of this one.